src/Persistence/Initialization/Updates/AddItemDropGroupForJewelsSeason6Update.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <copyright file="AddItemDropGroupForJewelsSeason6Update.cs" company="MUnique">
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace MUnique.OpenMU.Persistence.Initialization.Updates;
+
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using MUnique.OpenMU.DataModel.Configuration;
+using MUnique.OpenMU.PlugIns;
+
+/// <summary>
+/// This update creates a specific item drop group for jewels with a default chance of 5%.
+/// </summary>
+[PlugIn(PlugInName, PlugInDescription)]
+[Guid("F958CC5B-C1E6-4F67-B48D-4BF75EC5CAA8")]
+public class AddItemDropGroupForJewelsSeason6Update : AddItemDropGroupForJewels075Update
+{
+    /// <inheritdoc />
+    public override UpdateVersion Version => UpdateVersion.AddItemDropGroupForJewelsSeason6;
+
+    /// <inheritdoc />
+    public override string DataInitializationKey => VersionSeasonSix.DataInitialization.Id;
+
+    /// <inheritdoc />
+    protected override async ValueTask ApplyAsync(IContext context, GameConfiguration gameConfiguration)
+    {
+        await base.ApplyAsync(context, gameConfiguration).ConfigureAwait(false);
+        this.AddJewelToItemDrop(gameConfiguration, "Jewel of Creation");
+        this.AddJewelToItemDrop(gameConfiguration, "Jewel of Guardian");
```

Comment:
'Jewel of Guardian' only drops in Land of Trials, the map where only castle owners can grant access to.

Reply:
Previously it was enabled to be dropped by any mob with a level higher than 75.

I don't know if the only mobs that exceed that level are those on the Land of Trials map.

In any case, in my mind I have two options:
A) Leave the Jewel of Guardian in the jewel group, with the change I made so that only items from monsters with a sufficient level would drop, the same logic that was there before should be maintained.

https://github.com/MUnique/OpenMU/pull/474/files#diff-04390984d3c073558c83c0fa8e6c67c09da572f39be91c23e77c79cf9825c8d6R409

B) Remove the item from the jewel group, and add it to a new drop group for that specific map.

Reply:
I'd vote for B 😅

Reply:
Much clearer, really. So I'll prepare it :)

Reply:
@sven-n done!